src/services/news.service.ts

```diff
@@ -0,0 +1,51 @@
+import axios from "axios"
+import { format } from "date-fns"
+import { API_KEY } from "../api/api"
```

Comment:
Don't store the API key on a project file, instead use a `.env` file - [more info here](https://vitejs.dev/guide/env-and-mode.html)